recipes/thucyd/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "thucyd" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "e3c0c736746b0ac26b93fa8d2fcd2d90394adb080d60a84144aabf9f83325381"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - numpy >=1.14
+    - python >=3.5
+
+test:
+  imports:
+    - thucyd
+
+about:
+  home: "https://gitlab.com/thucyd-dev/thucyd"
+  license: Apache-2.0
+  license_family: "APACHE"
+  license_file: LICENSE
+  summary: "Scientific library for realtime signal processing and eigenanalysis of evolving systems."
```

The README is automatically generated by conda-smithy each time the package is re-rendered so you should never modify it directly. If you want to customise it you should modify the `about` second of the metadata, potentially including a `description`:

```suggestion
  summary: "Scientific library for realtime signal processing and eigenanalysis of evolving systems."
  description : |
     This awesome package does:
         * This
         * and this
         * and this

     If you use this for a publication please cite xxx.
```

---

Ok, got it. Thanks.